assets/emoticons.svg

```diff
@@ -0,0 +1,594 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
+   width="135.46666mm"
```

Comment:
Would be nice if the width and height were in pixel not in millimeter and match the resolution of the original image.
So 512 x 512 in this case.